src/cards/split.ts

```diff
@@ -49,6 +49,7 @@ function remove(unit: Unit.IUnit, underworld: Underworld) {
   unit.manaMax = manaMax;
   // Prevent unexpected overflow
   unit.mana = Math.min(manaMax, unit.mana);
+  unit.manaPerTurn = manaPerTurn;
```

Comment:
@SoulMuncher Do we want manaPerTurn cut in half when mana isn't? It'll make all the splits very slow to recover their mana.

Reply:
@jdoleary I thought mana was cut in half as well, so I'm not quite sure what you're asking.

Mana per turn is cut in half so that if you split a summoner or priest, it doesn't make them stronger by having two of them casting the spell at the normal speed. Cutting mana per turn was the solution we came up with to splitting a summoner and then the splits being completely unable to cast anything due to lower max mana than casting cost.

Reply:
Oh I see the costToCast remains the same and so does the max mana.
This might need to still be balanced, this means a split will make a summoner take 2x times as long to recharge mana, which is a long time.  Maybe that's the benefit of split,  but it feels like it might be OP for casters